values.yaml

```diff
@@ -245,6 +245,10 @@ global:
     # Only applicable if global.metrics.enabled is true.
     agentMetricsRetentionTime: 1m
 
+    # Configures mesh, ingress, and terminating gateways’
+    # Envoy metrics to Prometheus. Only applicable if global.metrics.enabled is true.
```

Comment:
I think it'd be good to expand here this a bit. Maybe something like:

If true, mesh, terminating, and ingress gateways will expose their Envoy metrics on port 20200 at the `/metrics` path and all gateway pods will have Prometheus scrape annotations. Only applicable if global.metrics.enabled is true.